proposals/wac-ucr/index.bs

```diff
@@ -12,35 +12,973 @@ Abstract:
+rushed immediately to the closest hospital in an ambulance.
+
+Oscar's government has recently rolled out a digital credential system
+for health care professionals, which is like a digital id card for
+doctors and care facilities that can be cryptographically verified.
+
+Oscar has an emergency care record at `https://oscar.example/emergency`, and
+Oscar's authorization system at `https://oscar.example` supports a
+verifiable credential authorization capability.
+
+Oscar has set permissions on `https://oscar.example/emergency` that grants
+someone in possession of a verifiable medical credential to have inherited
+read access to the contents. This gives them just enough background on
+his condition to treat him properly.
+
+### Possession of a link ### {#capabilities-link}
+
+Bob is about to give a confidential presentation to a group of his
+colleagues. His presentation is stored on his private server at
+`https://bob.example/presentation`.
+
+Bob is having trouble hooking his laptop up to the projector, and
+needs to present in just a few minutes. Anne presented before Bob,
+and offers to bring the presentation up for him on her system.
+
+Unfortunately, Anne doesn't have an identity ready to go that he can
+xauthorization. Luckily, his authorization system supports a link-based
+authorization capability.
+
+Bob quickly creates an authorization rule for `https://bob.example/presentation`
+that allows anyone in possession of a specially generated link to access
+the document with read-only permissions. He sets it to
+[[#conditional-time|expire]] in three hours. Bob gives the link to Anne
+and the presentation goes off perfectly.
+
+Conventional WAC Limitations {#limitations}
+================================================================================
+
+Conventional Web Access Control does not satisfy the following use cases:
+
+* [[#inheritance-adding]]
+* [[#inheritance-modifying]]
+* [[#inheritance-forcing]]
+* [[#conditional-time]]
+* [[#conditional-tag]]
+* [[#conditional-filter]]
+* [[#uc-applications]]
+* [[#uc-trustedissuers]]
+* [[#uc-validation]]
+* [[#capabilities-vc]]
+* [[#capabilities-link]]
 
 Definitions {#definitions}
-=====================
+================================================================================
+
+All definitions as stated below should be considered in the context of
+Web Access Control, whether explicitly stated or not.
+
+An <dfn>agent</dfn> is a distinct individual, group, organization, or piece
+of software with an [=identity=] that can be strongly authenticated.
 
-A <dfn>Definition</dfn> can be defined here and then referenced throughout
-the rest of the document.
+An <dfn>identity</dfn> is used to uniquely identify a given [=agent=],
+represented by a unique [[URI]]. Examples of compatible identity systems
+include [[WEBID]] and [[DID]].
+
+An <dfn>authenticated agent</dfn> is an [=agent=] that can prove control
+of a given [=identity=].
```

Comment:
@bblfish suggests "has proven control"

Reply:
Resolved in https://github.com/solid/authorization-panel/pull/84/commits/aa7984f25f101920d0db05d2ad2816c6293355b8